database/seeders/DatabaseSeeder.php

```diff
@@ -16,35 +13,6 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // Populate 5 users with 10 articles each other
-        User::factory(5)->create()->each(function ($user) {
-            $user->articles()->saveMany(Article::factory(10)->make());
-        });
-
-        // Populate categories
```

Comment:
It's better to separate this seeder to 2 seeders
1 - User with Articles
2 - Categories

and don't select all categories - just select fields you need = id